<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="imagesLayout">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGraphicsView" name="originalImageView">
          <property name="minimumSize">
           <size>
            <width>404</width>
            <height>260</height>
           </size>
          </property>
          <property name="foregroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGraphicsView" name="dissolveImageView">
          <property name="minimumSize">
           <size>
            <width>404</width>
            <height>260</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGraphicsView" name="modifiedImageView">
          <property name="minimumSize">
           <size>
            <width>404</width>
            <height>260</height>
           </size>
          </property>
          <property name="foregroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_20">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="instructionLabel">
          <property name="text">
           <string>Right click to bring up the popup menu to load an image or to select an operation</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_19">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="operationsLayout">
        <item>
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="quantizeLayout">
          <item>
           <widget class="QLabel" name="quantizeLabel">
            <property name="text">
             <string>Quantize</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="quantizeUnitLabel">
            <property name="text">
             <string>Levels</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="quantizeSpinBox">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="brightnessLayout">
          <item>
           <widget class="QLabel" name="brightenLabel">
            <property name="text">
             <string>Brighten</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="brightenUnitLabel">
            <property name="text">
             <string>Brightness Factor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="brightenSpinBox">
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="saturationLayout">
          <item>
           <widget class="QLabel" name="saturationLabel">
            <property name="text">
             <string>Saturation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="saturationUnitLabel">
            <property name="text">
             <string>Saturation Factor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="saturationSpinBox">
            <property name="minimum">
             <number>-300</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_17">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="scaleLayout">
          <item>
           <widget class="QLabel" name="scaleLabel">
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="scaleUnitLabel">
            <property name="text">
             <string>Scale Factor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="scaleSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="rotateLayout">
          <item>
           <widget class="QLabel" name="rotateLabel">
            <property name="text">
             <string>Rotate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="rotateUnitLabel">
            <property name="text">
             <string>Degrees</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rotateSpinBox">
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="contrastLayout">
          <item>
           <widget class="QLabel" name="contrastLabel">
            <property name="text">
             <string>Contrast</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="contrastUnitLabel">
            <property name="text">
             <string>Contrast Factor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="contrastSpinBox">
            <property name="minimum">
             <number>-300</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="dissolveLayout">
          <item>
           <widget class="QLabel" name="dissolveLabel">
            <property name="text">
             <string>Dissolve</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dissolveUnitLabel">
            <property name="text">
             <string>Parametric Distance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="dissolveSpinBox">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
